packages/devextreme/scss/widgets/fluent/scheduler/_index.scss

```diff
@@ -220,9 +220,26 @@ $fluent-scheduler-agenda-time-panel-cell-padding: 8px;
 
 
   &.dx-scheduler-header-panel-current-time-cell {
+    color: $scheduler-panel-text-color;
+
     &::before {
       content: none;
     }
+
+    > div:last-child {
```

Comment:
Does it really make sense to have such a strict selector?

Reply:
Not really. Refactored.